Jenkinsfile.nightly

```diff
@@ -110,6 +112,21 @@ pipeline {
                 }
             }
         }
+*/
+        stage('Nightly-Docker-Image') {
+            //when {
+            //    branch 'master'
+            //}
+            steps {
+                withDockerRegistry([ credentialsId: "NIGHTLY_DOCKER_HUB_CRED", url: "" ]) {
```

Comment:
Out of curiosity, where does NIGHTLY_DOCKER_HUB_CRED come from? I couldn't find it in https://github.com/cilium/Jenkins-library or in cilium

Reply:
It's stored as a jenkins credential, we don't store it in repo for obvious reasons :D